nameserver/http.go

```diff
@@ -35,6 +35,11 @@ func (n *Nameserver) HandleHTTP(router *mux.Router, dockerCli *docker.Client) {
 			return
 		}
 
+		if !dns.IsSubDomain(n.domain, hostname) {
+			n.infof("Ignoring name %s, not in %s", hostname, n.domain)
```

Comment:
The message is identical to the one used in 1.0. More context won't hurt though!